utils/warnings.ml

```diff
@@ -383,7 +389,8 @@ let parse_opt error active errflag s =
        loop (i+1)
     | _ -> error ()
   in
-  loop 0
+  loop 0;
+  errors.(69) <- false (* Missed potential trmc call is not an error *)
```

Comment:
I am unfamiliar with the code so I am not sure what happens here, but it looks smelly.  What is special about this warning compared to all the other warnings which are not errors, and why does this needs to be hard-coded?

Reply:
I think this was a quick hack from the setup commit, meant to avoid breaking user code with this warning even if they unreasonably set all warnings as errors. I removed this change. Thanks!